user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/Ad_hoc_data/Ad_hoc_Tutorials_Satellites.md

```diff
@@ -0,0 +1,89 @@
+---
+uid: Ad_hoc_Tutorials_Satellites
+---
+
+# Building a GQI data source that fetches satellites
+
+In this tutorial, you will learn how you can create a GQI data source that can be used to add ad hoc data to a dashboard or low-code app.
+
+Expected duration: 15 minutes.
+
+## Prerequisites
+
+- [Visual Studio](https://visualstudio.microsoft.com/downloads/) with [DataMiner Integration Studio](xref:Installing_and_configuring_DataMiner_Integration_Studio)
+- A DataMiner Agent [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud)
+
+## Overview
+
+- [Step 1: Get a quick start from the catalog](#step-1-get-a-quick-start-from-the-catalog)
+- [Step 2: Open the data source in Visual Studio](#step-2-open-the-data-source-in-visual-studio)
+- [Step 3: Complete the data source](#step-3-complete-the-data-source)
+- [Step 4: Use the data source](#step-4-use-the-data-source)
+
+## Step 1: Get a quick start from the catalog
+
+1. Go to [https://catalog.dataminer.services/catalog/5406](https://catalog.dataminer.services/catalog/5406)
+
+1. Deploy the catalog item to your DataMiner Agent.
```

Comment:
Could you make this description a bit more detailed, so it's definitely clear how this is done?